CMSgov/mint-app
# [EASI-3947][EASI-4024] Notification Center Feature -- Shared Model Plan

# EASI-3947, EASI-4024



## Changes and Description

- BE: see https://github.com/CMSgov/mint-app/pull/1013
- FE: see https://github.com/CMSgov/mint-app/pull/1026



## How to test this change

1. High overview functionality check
2. Share model plan with someone else and check that someone else sees the notification in their notification center and email




## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
